rust/src/lib/ip.rs

```diff
@@ -297,6 +307,17 @@ impl InterfaceIpv4 {
                     }
                 }
             }
+            if let Some(forwarding) = self.forwarding {
+                if !self.enabled {
+                    if is_desired {
+                        log::warn!(
+                            "Ignoring `forwarding: {}` as IPv4 is disabled",
+                            forwarding
+                        );
+                    }
+                    self.forwarding = None;
+                }
+            }
```

Comment:
This is inside the `if  self.is_auto() {}` so it will be ignored for manual IP config. Additionally, we don't do this check for other properties, so I'd say that it's better not to do it for this one.

Reply:
We should not raise warning when `forwarding: false` with `enabled: false`.

Reply:
I changed it to raise a warning when setting the `forwarding: false` with `enabled: false`.